### PR TITLE
VTCCA-0 seperating out the agent call that require permission details…

### PR DESCRIPTION
**Pull request by DanielConnelly**
… from those that do. (as these fields are not part of the schema)

### CODE REVIEW

File: app/connectors/PropertyRepresentationConnector.scala [hunk 1]
```diff
@@ -52,15 +52,12 @@ class PropertyRepresentationConnector @Inject()(
...
-      s"&size=${params.pageSize}" +
-      s"&representationStatus=PENDING"
+    val url = s"$baseUrl/authorisation-search-api/agents/$organisationId/authorisations"
     
-    http.GET[AgentAuthResultBE](url).map(x => {
+    http.GET[AgentAuthResultBE](url, Seq("start" -> params.startPoint.toString, "size" -> params.pageSize.toString, "representationStatus" -> "PENDING")).map(x => {
       PropertyRepresentations(x.filterTotal, x.authorisations.map(_.toPropertyRepresentation))
     })
-  }
+  }git
```
Comment (habeeb01) [inline app/connectors/PropertyRepresentationConnector.scala]: you might have a compilation error. "git" ??

File: conf/logback.xml [hunk 2]
```diff
@@ -45,7 +45,7 @@
 
     <logger name="uk.gov" level="DEBUG"/>
 
-    <logger name="application" level="DEBUG"/>
+    <logger name="application" level="TRACE"/>
```
Comment (habeeb01) [inline conf/logback.xml]: why is this changing?

Comment (DanielConnelly) [reply conf/logback.xml]: local testing changes good catch.

File: app/connectors/PropertyRepresentationConnector.scala (hunk 1 above)
Comment (DanielConnelly) [reply app/connectors/PropertyRepresentationConnector.scala]: local testing changes good catch.

**Pull request merged**